src/components/heading/demo/weight.twig

```diff
@@ -1,11 +1,7 @@
-{% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
-  class: 'o-rhythm--condensed'
-} only %}
-  {% block content %}
-      {% include '@cloudfour/components/heading/heading.twig' with {
-        level: 1,
-        weight: 'light',
-        content: 'c-heading--weight-' ~ 'light'
-      } only %}
-  {% endblock %}
-{% endembed %}
+{% block content %}
+    {% include '@cloudfour/components/heading/heading.twig' with {
+      level: 1,
+      weight: 'light',
+      content: 'c-heading--light'
+    } only %}
+{% endblock %}
```

Comment:
I don't know if this demo file is needed at all?

In your stories file, you should be able to import the template directly...

```js
import template from './heading.twig';
```

And then your story can pass along those args (indentation may be a little off here)...

```jsx
<Canvas>
  <Story name="Light weight" args={{ level: 1, weight: 'light', content: 'c-heading--light' }}>
  { (args) => template(args) }
  </Story>
</Canvas>
```